.github/workflows/run_tests/action.yml

```
@@ -11,7 +11,7 @@ runs:
     - name: Install lancedb
       shell: bash
       run: |
-        pip3 install $(ls target/wheels/lancedb-*.whl)[tests,dev]
+        pip3 install $(ls target/wheels/lancedb-*.whl)[tests,dev] tantivy
```

We are selecting the `tests` and `dev` dependencies are part of the install. Could you instead add `tantivy` to `tests` dependencies?
```suggestion
        pip3 install $(ls target/wheels/lancedb-*.whl)[tests,dev]
```

It would go here:
https://github.com/lancedb/lancedb/blob/8a1227030a27dd229c2492e534682d4eae1f3cf4/python/pyproject.toml#L50-L59